src/renderers/webxr/WebXRManager.js

```diff
@@ -228,6 +228,19 @@ class WebXRManager extends EventDispatcher {
 
 		this.setSession = async function ( value ) {
 
+			if ( session ) {
```

Comment:
Without this we would have multiple event listeners on session restart. I guess there is no harm trying to remove listeners which are not added already on first session.

Would it make more sense to do this in `onSessionEnd`?

Reply:
Makes sense!